library/src/scala/tasty/reflect/SymbolOps.scala

```diff
@@ -9,6 +9,17 @@ trait SymbolOps extends Core { selfSymbolOps: FlagsOps =>
     def classSymbol(fullName: String)(given ctx: Context): Symbol =
       internal.Symbol_of(fullName)
 
+    /** Generates a new method symbol with the given parent, name and type.
+     *
+     *  This symbol starts without an accompanying definition.
+     *  It is the meta-programmer's responsibility to provide exactly one corresponding definition by passing
+     *  this symbol to the DefDef constructor.
+     *
+     *  @note As a macro can only splice code into the point at which it is expanded, all generated symbols must be
+     *        direct or indirect children of the reflection context's owner. */
+    def newMethod(parent: Symbol, name: String, tpe: Type, flags: Flags = Flags.EmptyFlags, privateWithin: Option[Symbol] = None)(given ctx: Context): Symbol =
```

Comment:
We are trying to avoid default parameters for compatibility reasons. Remove them and maybe add an overloaded version without the last two.
```scala
    /** ... */
    def newMethod(parent: Symbol, name: String, tpe: Type)(given ctx: Context): Symbol =
      newMethod(parent, name, tpe, Flags.EmptyFlags, None)

    /** ... */
    def newMethod(parent: Symbol, name: String, tpe: Type, flags: Flags, privateWithin: Option[Symbol])(given ctx: Context): Symbol =
```